app/views/works/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<main>
+<h2><%= @work.title %></h2>
+<p>Created by:  <%= @work.creator %></p>
+<p>Published: <%= @work.publication_year %></p>
+<p>Description: <%= @work.description %></p>
+
+  <section>
+    <%= link_to "Back to media ranks", root_path, class: "btn btn-primary" %>
+    <%= link_to "Edit", edit_work_path(@work), class: "btn btn-primary" %>
+    <%= link_to "Upvote", work_votes_path(@work), method: :post, class: "btn btn-primary" %>
+    <%= link_to "Delete", work_path(@work), method: :delete, class: "btn btn-danger", data: {confirm: "Are you sure?"}%>
+  </section>
+
+  <section class="works-votes__container">
+    <h3 class="works-votes__header"> <%= @work.votes.count %> Votes for this <%= @work.category.capitalize %></h3>
+
+    <table class="table">
+      <thead>
+      <tr>
+        <th width = 50%>User</th>
```

Comment:
In the future you should avoid inline styling like this.  (The width should be set in the CSS.)